job-scheduler/src/main/java/uk/gov/hmcts/cmc/scheduler/jobs/NotificationEmailJob.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.cmc.scheduler.jobs;
+
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.scheduler.services.NotificationEmailService;
+
+import java.text.SimpleDateFormat;
+import java.time.ZonedDateTime;
+
+@Component
+public class NotificationEmailJob implements Job {
+
+    private static final Logger logger = LoggerFactory.getLogger(NotificationEmailJob.class);
+
+    @Autowired
+    private NotificationEmailService notificationEmailService;
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
```

Comment:
this job should probably be removed in the initial PR, / add a job that only logs and doesn't send emails